Composer/packages/client/src/components/CreationFlow/v2/CreationFlow.tsx

```diff
@@ -150,13 +154,21 @@ const CreationFlowV2: React.FC<CreationFlowProps> = () => {
     saveProjectAs(projectId, formData.name, formData.description, formData.location);
   };
 
+  const handleMigrate = (formData) => {
+    handleDismiss();
+    setCreationFlowStatus(CreationFlowStatus.MIGRATE);
+    migrateProjectTo(projectId, formData.name, formData.description, formData.location);
```

Comment:
If formData has type any, do you want to check name, description, and/or location have values before calling migrateProjectTo?